src/modules/rollenmapping/api/rollenmapping.controller.ts

```diff
@@ -0,0 +1,173 @@
+import {
+    ApiBadRequestResponse,
+    ApiBearerAuth,
+    ApiForbiddenResponse,
+    ApiInternalServerErrorResponse,
+    ApiNotFoundResponse,
+    ApiOAuth2,
+    ApiOkResponse,
+    ApiTags,
+    ApiUnauthorizedResponse,
+} from '@nestjs/swagger';
+import {
+    Controller,
+    Delete,
+    ForbiddenException,
+    Get,
+    NotFoundException,
+    Param,
+    Post,
+    Put,
+    Query,
+    UseFilters,
+} from '@nestjs/common';
+import { RollenMapping } from '../domain/rollenmapping.js';
+import { RollenMappingRepo } from '../repo/rollenmapping.repo.js';
+import { Permissions } from '../../authentication/api/permissions.decorator.js';
+import { ClassLogger } from '../../../core/logging/class-logger.js';
+import { RollenSystemRecht } from '../../rolle/domain/rolle.enums.js';
+import { PersonPermissions } from '../../authentication/domain/person-permissions.js';
+import { RollenMappingFactory } from '../domain/rollenmapping.factory.js';
+import { SchulConnexValidationErrorFilter } from '../../../shared/error/schulconnex-validation-error.filter.js';
+import { RollenmappingCreateBodyParams as RollenmappingCreateBodyParams } from './rollenmapping-create-body.params.js';
+
+@UseFilters(new SchulConnexValidationErrorFilter())
+@ApiTags('rollenmapping')
+@ApiBearerAuth()
+@ApiOAuth2(['openid'])
+@Controller({ path: 'rollenmapping' })
+export class RollenmappingController {
+    public constructor(
+        private readonly rollenMappingRepo: RollenMappingRepo,
+        private readonly rollenMappingFactory: RollenMappingFactory,
+        private readonly logger: ClassLogger,
+    ) {}
+
+    @Get(':id')
+    @ApiOkResponse({ description: 'The rollenmapping was successfully returned', type: RollenMapping })
+    @ApiUnauthorizedResponse({ description: 'Unauthorized to retrieve the rollenmapping' })
+    @ApiNotFoundResponse({ description: 'No rollenmapping found' })
+    @ApiForbiddenResponse({ description: 'Insufficient rights to retrieve the rollenmapping' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error while retrieving the rollenmapping' })
+    public async getRollenmappingWithId(
+        @Param('id') id: string,
+        @Permissions() personPermission: PersonPermissions,
+    ): Promise<RollenMapping<true>> {
+        if (!(await personPermission.hasSystemrechteAtRootOrganisation([RollenSystemRecht.ROLLEN_VERWALTEN]))) {
```

Comment:
This should additionally check, if the user also has access to the LMS this Rollenmapping is associated with